Corona-Warn-App/src/main/res/layout/home_create_trace_location_card.xml

```diff
@@ -30,26 +28,36 @@
         android:text="@string/create_trace_location_card_subtitle"
         app:layout_constraintEnd_toStartOf="@id/create_trace_location_card_icon"
         app:layout_constraintStart_toStartOf="@id/create_trace_location_card_headline"
-        app:layout_constraintTop_toBottomOf="@id/create_trace_location_card_headline" />
+        app:layout_constraintTop_toBottomOf="@id/create_trace_location_card_headline"
+        tools:text="@string/create_trace_location_card_subtitle" />
 
     <ImageView
```

Comment:
Afaik we are trying to streamline every image view to `<androidx.appcompat.widget.AppCompatImageView`
Would be great to for this small refactoring in your ticket 🤖

Reply:
@AlexanderAlferov 
thx for the info. adjusted

Reply:
@AlexanderAlferov As I understood in https://github.com/corona-warn-app/cwa-app-android/pull/2700#pullrequestreview-622693686, writing `ImageView` is the same as writing `androidx.appcompat.widget.AppCompatImageView`. The only thing that needs to be looked out for is using `android.widget.ImageView`, as that is not the same and won't make use of the compat libraries.